about/complying_with_licenses.rst

```diff
@@ -22,9 +27,9 @@ with the original one.
     If you are interested in licence compliance as a contributor, you can find
     guidelines :ref:`here <doc_best_practices_for_engine_contributors_license_compliance>`.
 
-.. warning::
+.. tip::
 
-    In your project's credits screen, remember to also list third-party notices
+    Alongside the Godot license text, remember to also list third-party notices
     for assets you're using, such as textures, models, sounds, music and fonts.
 
     Free assets in particular often come with licenses that require attribution.
```

Comment:
I wonder whether this line is necessary, as in, why particularly highlight free assets? I think all assets used should be checked. It seems to me that the previous paragraph ("remember to also list third-party notices") is all that's required here.

Reply:
See the line below and it becomes obvious 🙂

Reply:
I'll rephrase a bit.